internal/clearingway/guild.go

```diff
@@ -54,52 +54,54 @@ func (g *Guild) Init(c *ConfigGuild) {
 		g.Encounters.Encounters = append(g.Encounters.Encounters, encounter)
 	}
 
-	if c.ConfigRoles != nil && c.ConfigRoles.RelevantParsing == false {
-		g.RelevantParsingEnabled = false
-	} else {
-		g.RelevantParsingEnabled = true
-	}
+	if c.ConfigRoles != nil {
+		if c.ConfigRoles.RelevantParsing == false {
+			g.RelevantParsingEnabled = false
+		} else {
+			g.RelevantParsingEnabled = true
+		}
```

Comment:
are there any particular reasons why we need to have this if else block if we're just going to assign the bool to config values?

Reply:
The default value I think. So if `c.ConfigRoles.RelevantParsing` is nil, then `RelevantParsingEnabled` will be true. This is very confusing though and the whole thing probably needs to be rethought.